snmp/datadog_checks/snmp/data/default_profiles/netgear-generic.yaml

```diff
@@ -0,0 +1,11 @@
+extends:
```

Comment:
Shall we rename netgear-generic.yaml to netgear.yaml ? to be consistent with the other profiles that are named <VENDOR>.yaml